sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-data.js

```diff
@@ -118,18 +144,20 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
             // Check the top pixel and bottom pixel and make sure they have
             // the right color.
-            debug("Checking bottom pixel");
-            wtu.checkCanvasRect(gl, 0, 0, 1, 1, bottomColor, "shouldBe " + bottomColor);
-            debug("Checking top pixel");
-            wtu.checkCanvasRect(gl, 0, 1, 1, 1, topColor, "shouldBe " + topColor);
+            debug("Checking " + (flipY ? "top" : "bottom"));
+            wtu.checkCanvasRect(gl, 0, bottom, halfWidth, halfHeight, tl, "shouldBe " + tl);
```

Comment:
This is also problematic.  We can't use RGBA/UNSIGNED_BYTE for readPixels() if it's INTEGER type color image.  We will have to use RGBA/INTEGER.